packages/dd-trace/src/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const { isFalse } = require('./util')
+
+module.exports = isFalse(process.env.DD_TRACE_ENABLED)
+  ? require('./noop/proxy')
```

Comment:
@juan-fernandez We could do something similar for https://github.com/DataDog/dd-trace-js/issues/2297